ApeWorX/ape-vyper
# feat: `trace_source` implementation [APE-754]

### What I did

Implements necessary API methods for performing traceback line showings as well as coverage.

### How I did it

Implementation borrowed from Brownie somewhat, but basically track the trace while mapping PC to known locations, including both from the source code as well as from the error map.

### How to verify it

Use this PR from ape core: https://github.com/ApeWorX/ape/pull/1337 and cause uncaught contract logic errors to happen of any kind.
Observe them in the traceback.

### Checklist
